### DISCUSSION TITLE
Does deegree OGC API supports the requirements class "INSPIRE-CRS"?

### DISCUSSION THREAD

**Discussion by tfr42** [deegree/deegree-ogcapi#67] [Q&A] [answered]
Does deegree OGC API supports the requirements class "INSPIRE-CRS" as described in ["Setting up an INSPIRE Download service based on the OGC API-Features standard"](https://github.com/INSPIRE-MIF/gp-ogc-api-features/blob/master/spec/oapif-inspire-download.md)?

**Comment by Kate-Lyndegaard** [comment, marked as answer]
AFAIK the requirements class contains one recommendation: to include at least one coordinate system from the given list. As this is configurable, I would suggest that it is supported.

**Reply by tfr42** [reply]
Requirement class  http://inspire.ec.europa.eu/id/spec/oapif-download/1.0/req/inspire-crs is supported.
